<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>812</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AWS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>620</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_path">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/경로선택.png</normaloff>
         <disabledon>:/button/경로선택2.png</disabledon>:/button/경로선택.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_1st">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/1st.png</normaloff>
         <disabledoff>:/button/1st2.png</disabledoff>:/button/1st.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_learning">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/learning.png</normaloff>
         <disabledoff>:/button/learning2.png</disabledoff>:/button/learning.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/run.png</normaloff>
         <normalon>:/button/run2.png</normalon>
         <disabledoff>:/button/run2.png</disabledoff>:/button/run.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/stop.png</normaloff>
         <disabledoff>:/button/stop2.png</disabledoff>:/button/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_turnoff">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/button/turnoff.png</normaloff>
         <normalon>:/button/turnoff2.png</normalon>
         <disabledoff>:/button/turnoff2.png</disabledoff>
         <disabledon>:/button/turnoff2.png</disabledon>:/button/turnoff.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="AWS_logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon/drone.qrc">:/drone/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="warehouse_Name">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 182, 255);
font: 18pt &quot;HY중고딕&quot;;
</string>
    </property>
    <property name="text">
     <string notr="true">테스트 창고</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>214</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_drone" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btn_drone">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/drone_label/drone_label.png</normaloff>
         <normalon>:/drone_label/drone_label_clicked.png</normalon>:/drone_label/drone_label.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">dronegroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_drone_plus">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="icon/drone.qrc">
         <normaloff>:/drone_label/drone_pluselabel.png</normaloff>
         <normalon>:/drone_label/drone_pluselabel_clicked.png</normalon>:/drone_label/drone_pluselabel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">dronegroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QOpenGLWidget" name="batteryGuage">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>530</y>
      <width>260</width>
      <height>260</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>320</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string notr="true">GOODS</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>100</green>
        <blue>100</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">EAC</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string notr="true">하이하이루</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string notr="true">하이</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsSelectable</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="video_frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>680</width>
      <height>490</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon/drone.qrc">:/drone/그림1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tempMap">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0.5px solid rgb(78, 182, 255);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>182</red>
        <green>78</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="firstlabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>320</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0.5px solid rgb(78, 182, 255);
border-radius: 5px;
color : rgb(78, 182, 255);
</string>
    </property>
    <property name="text">
     <string>제품 파악 목록</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1359</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon/drone.qrc">:/background/bgImg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setting">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 182, 255);</string>
    </property>
    <property name="text">
     <string notr="true">세팅버튼</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tableWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>AWS_logo</zorder>
   <zorder>warehouse_Name</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>batteryGuage</zorder>
   <zorder>video_frame</zorder>
   <zorder>tempMap</zorder>
   <zorder>firstlabel</zorder>
   <zorder>btn_setting</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_start</tabstop>
 </tabstops>
 <resources>
  <include location="icon/drone.qrc"/>
  <include location="../../.designer/backup/icon/drone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_drone_plus</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_drone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_drone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>734</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_turnoff</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>turnOffAWS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>734</y>
    </hint>
    <hint type="destinationlabel">
     <x>994</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_learning</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_learning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_setting</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>temp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop_drone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>867</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>848</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_1st</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>test_drone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>711</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>744</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_path</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>drone_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>706</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <signal>setColor()</signal>
  <slot>temp()</slot>
  <slot>add_drone()</slot>
  <slot>start_drone()</slot>
  <slot>slot1()</slot>
  <slot>turnOffAWS()</slot>
  <slot>start_learning()</slot>
  <slot>stop_drone()</slot>
  <slot>test_drone()</slot>
  <slot>drone_path()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="dronegroup"/>
 </buttongroups>
</ui>
